sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/autorest.md

```diff
@@ -0,0 +1,23 @@
+# Azure.Iot.TimeSeriesInsights
+
+## Azure Time Series Insights swagger
+
+## Official swagger document
+
+The official swagger specification for Azure Time Series Insights can be found [here](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/9a6510d0597a55d39ab1edcf22abab4631cbc0d3/specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/timeseriesinsights.json).
+
+## Code generation
+
+Run `generate.ps1` in this directory to generate the code.
+
+## AutoRest Configuration
+
+> see <https://aka.ms/autorest>
+
+``` yaml
+#when generating from official source - The raw link must have a commit hash for C# generator
+#input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/9a6510d0597a55d39ab1edcf22abab4631cbc0d3/specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/timeseriesinsights.json
+
+#if you want to generate using local swagger copy:
```

Comment:
I noticed we are not setting the namespace for the generated code in ADT either ... are we happy with the generated namespace?

Reply:
The namespace that is generated right now is:
Azure.Iot.TimeSeriesInsights

We think it's fine, but we'll get feedback from them if they want to name it something else.